File: text-translation-demo/csharp/TextTranslationDemo/TextTranslationOption.cs
using common;

namespace TextTranslationDemo
{
    public class TextTranslationOption 
    {
        public int FieldId { get; set; }
        public string TargetLang { get; set; }
        public string SourceLang { get; set; }
        public bool? AutoSplit { get; set; }
        public TranslationEngineType Type { get; set; }
        public bool? RemoveFakeLineBreak { get; set; }
    }
}

File: text-translation-demo/vb.net/TextTranslationDemo/TextTranslationOption.vb
Imports common.common

Namespace TextTranslationDemo
    Public Class TextTranslationOption
        Public Property FieldId As Integer
        Public Property TargetLang As String
        Public Property SourceLang As String
        Public Property AutoSplit As Boolean?
        Public Property Type As TranslationEngineType
        Public Property RemoveFakeLineBreak As Boolean?
    End Class
End Namespace
